### ISSUE TITLE
New Markets & Native APYs

### ISSUE THREAD

**Issue by defi-er** [dev-protocol/DIPs#40]
dip: <#37>
title: <New Markets & Native APYs>
status: WIP
author: Scott G

created: 2021-01-05

Simple Summary

This proposal calls for New Markets (example: Research) to have their own native APY based on DEV staked and assets onboarded.

Abstract

Currently, if a New Market is added to Dev Protocol the assets are lumped together. For a Market's creators this is detrimental because the work of one sector is not equivalent to the work of another. For example, a Youtube content creator vs. an OSS developer. Each Market should have its own Native APY to provide so that the APY places a value on the Market's creator's work. This proposal also prevents the devaluation of work from different Markets, increases Market health, and creates Market APY arbitrage opportunity. 

Motivation

This proposal also prevents the devaluation of work from different Markets, increases Market health, and creates Market APY arbitrage opportunity. 

Specification

Overview

The feature will be implemented by creating a fork of the Dev Protocol for each New Market.

Copyright

Copyright and related rights waived via CC0.

**Comment by calaber24p** [comment]
How do we implement this without increasing inflation by a significant amount? Would each market have a new coin associated with them ? I personally think we should keep the markets tied together and make adjustments to the inflation schedule if we find APY is too low to entice stakers.

Separating markets would also have the effect of making certain markets more profitable and have them compete against each other. Ultimately this forces people to support projects they might not have supported in order to chase yield. I am not against the idea completely, I would just have to see how the math works out before I can have a fully fledged opinion.

**Comment by defi-er** [comment]
Dev Protocol's value proposition is that there is **yield for supporting open assets**. Yield acts as an incentive to bring money into the protocol. This value proposition is improved by this proposal. The current model is detrimental as it devalues a creator's work by lumping uncorrelated markets together. It's equivalent to valuing an artist relative to the work of Linux Foundation. Markets should have their own APYs that represent the value patrons place on their sector. 

_**The DIP I proposed takes Dev Protocol a step forward to becoming an ETF provider for Open Assets in which different ETFs are made that require the DEV token to participate.**_

**Comment by calaber24p** [comment]
> **Color for Dev Community**
> 
> I'll tell you how Dev Protocol works, but you can also find this information at [Docs](docs.devprotocol.xyz).
> 
> **How Dev Protocol's Inflation Works**
> Inflation is a counter-balance between assets onboarded and DEV staked. If one asset is onboarded then there's .00012 DEV minted per second which is minimal inflation. New Markets under this DIP would have significant inflation ONLY if assets outpace > DEV staked.

What I am asking is how do we set separate native APYs for different markets under your plan? The creation of a new market will increase the issuance but the APY shouldn't be different under the current system from market to market. 

My original point still stands where markets should be tied together and provide the same APY. If we have them compete and offer arbitrage opportunities between them, we might be giving value to a market that wouldn't have it otherwise. We might artificially prop up the support of a market through this system. I still believe new markets should be used only to separate the content and not governed under a new system.

**Comment by aggre** [comment]
I also think we need to discuss how to determine the native APYs.

Let's continue those discussions in #41. I have some ideas too.

**Comment by defi-er** [comment]
> If we have them compete and offer arbitrage opportunities between them, we might be giving value to a market that wouldn't have it otherwise. We might artificially prop up the support of a market through this system. I still believe new markets should be used only to separate the content and not governed under a new system.

This is how efficient markets work. Yield is competitive.

**Comment by Akira-Taniguchi** [comment]
I'm conditionally in favor of the idea of changing the prorating ratio by property.
　If there is already some experience (e.g. 10000DEVmint), it may be possible to allow some changes.
　In that case, the authority to change is the author.
　If the user has been staking for a long time, it may be necessary to add some special benefits.

**Comment by Cold-A-Muse** [comment]
I agree with @defi-er here. Yield should be competitive and different per market. If my APY as a nuclear scientist decreases because someone is making paintings I am not sure how I would feel about that and seems strange. If some other scientist gets a lot of funding and therefor my apy decreases sounds a lot more logical

**Comment by aggre** [comment]
Thank you for your views! This issue has been moved to #41. Also, I would like to consolidate the same topics into this issue: https://github.com/dev-protocol/DIPs/issues/48
